kahuna/public/js/search/query.html

```diff
@@ -62,7 +62,14 @@
                                gu:preset-dates="searchQuery.sinceOptions"
                                gu:first-day="1">
                 </gu-date-range>
-            <li>
+            </li>
+
+            <li class="search__filter-item">
+                <select class="search__filter"
+                        ng:model="searchQuery.filter.orderBy"
+                        ng:options="sortOrder.value as sortOrder.label for sortOrder in searchQuery.orderByOptions">
+                </select>
+            </li>
```

Comment:
I wonder if having it in the search filters bar and right next to the date range picker could be misinterpreted by users in thinking that:

1. It's a search filter
2. It applies to the date range

It's also worth noting there isn't any hint that this is about ordering.

Would it help to add a label and perhaps move it away from the filters and the date range picker to avoid confusion? As a "post filter" and more advanced option, could it live closer to the results, by the result count?

cc @alastairjardine